web/site/js/vis/Node.js

```diff
@@ -140,12 +139,35 @@ export default class Node extends VObject {
     }
 
     _createStatusSprite() {
+        return new PIXI.Sprite(this._getStatusSpriteTexture());
+    }
+
+    _createPartitionSprite() {
+        return new PIXI.Sprite(this._getPartitionSpriteTexture());
+    }
+
+    _getStatusSpriteTexture() {
+        switch (this.role) {
+            case OtDeviceRole.OT_DEVICE_ROLE_LEADER:
+            case OtDeviceRole.OT_DEVICE_ROLE_ROUTER:
+                return Resources().WhiteSolidHexagon64.texture;
```

Comment:
I feel that the Hexagon looks like it's a little smaller than Circle. Do you agree?
Should we increase the scale for Leader/Router for a little bit? @ctan-g @qilinkang

Reply:
Maybe increase the radius to `22`?

Reply:
Can you try and find a better radius? @ctan-g

Reply:
`22` seems to make them look consistent.

Reply:
Can you update the screenshot ?

Reply:
Updated

Reply:
Thanks, it looks better now.